app/tests/spec/views/settings/recovery_codes.js

```diff
@@ -209,4 +213,19 @@ describe('views/settings/recovery_codes', () => {
       assert.equal(view.$('.graphic-recovery-codes-download').length, 0);
     });
   });
+
+  describe('should continue sign-in after generating from low recovery codes', () => {
+    beforeEach(() => {
+      view.model.set('previousViewName', 'sign_in_recovery_code');
+      sinon.spy(view, 'invokeBrokerMethod');
+      return view.$('.two-step-authentication-done').click();
+    });
+
+    it('call correct broker method', () => {
+      const args = view.invokeBrokerMethod.args[0];
```

Comment:
Is it worth asserting the value of `view.invokeBrokerMethod.callCount` before trying to dereference `args`? That might make the error a bit clearer if it ever stops being called and the test starts failing?